### ISSUE TITLE
std: node compatibility issue with stream/web does not provide export named 'ReadableStreamBYOBReader'

### ISSUE THREAD

**Issue by Antonio476587** [denoland/deno#17752]
``` bash
# deno run -A npm:astro build
 error   The requested module 'node:stream/web' does not provide an export named 'ReadableStreamBYOBReader'
SyntaxError: The requested module 'node:stream/web' does not provide an export named 'ReadableStreamBYOBReader'
    at async runCommand (file:///home/seo/.cache/deno/npm/registry.npmjs.org/astro/2.0.4/dist/cli/index.js:156:34)
    at async cli (file:///home/seo/.cache/deno/npm/registry.npmjs.org/astro/2.0.4/dist/cli/index.js:180:5)
```

https://deno.land/std@0.177.0/node/stream/web.ts?source :
``` typescript
// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
const _ByteLengthQueuingStrategy = ByteLengthQueuingStrategy,  _CountQueuingStrategy = CountQueuingStrategy,  _ReadableByteStreamController = ReadableByteStreamController,  _ReadableStream = ReadableStream,  _ReadableStreamDefaultController = ReadableStreamDefaultController,  _ReadableStreamDefaultReader = ReadableStreamDefaultReader,  _TextDecoderStream = TextDecoderStream,  _TextEncoderStream = TextEncoderStream,  _TransformStream = TransformStream,  _TransformStreamDefaultController = TransformStreamDefaultController,  _WritableStream = WritableStream,  _WritableStreamDefaultController = WritableStreamDefaultController,  _WritableStreamDefaultWriter = WritableStreamDefaultWriter;
export {  _ByteLengthQueuingStrategy as ByteLengthQueuingStrategy,  _CountQueuingStrategy as CountQueuingStrategy,  _ReadableByteStreamController as ReadableByteStreamController,  _ReadableStream as ReadableStream,  _ReadableStreamDefaultController as ReadableStreamDefaultController,  _ReadableStreamDefaultReader as ReadableStreamDefaultReader,  _TextDecoderStream as TextDecoderStream,  _TextEncoderStream as TextEncoderStream,  _TransformStream as TransformStream,  _TransformStreamDefaultController as TransformStreamDefaultController,  _WritableStream as WritableStream,  _WritableStreamDefaultController as WritableStreamDefaultController,  _WritableStreamDefaultWriter as WritableStreamDefaultWriter,};
export default {  ReadableStream,  ReadableStreamDefaultReader,  ReadableByteStreamController,  ReadableStreamDefaultController,  TransformStream,  TransformStreamDefaultController,  WritableStream,  WritableStreamDefaultWriter,  WritableStreamDefaultController,  ByteLengthQueuingStrategy,  CountQueuingStrategy,  TextEncoderStream,  TextDecoderStream,};

```

**Comment by bartlomieju** [comment]
This should be as simple as updating the exports in `ext/node/polyfills/stream/web.ts`

**Comment by Antonio476587** [comment]
I would like to make it

**Comment by aapoalas** [comment]
> I would like to make it

Go ahead, and thank you!

**Comment by bartlomieju** [comment]
Fixed in #17803
